### PR TITLE
Codacy PR comment test

### PR DESCRIPTION
**Pull request by rodrigok**
Closes #(insert issue number)

### CODE REVIEW

File: packages/rocketchat-lib/server/functions/setEmail.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 RocketChat._setEmail = function(userId, email) {
 	email = s.trim(email);
-	if (!userId) {
+	if (!userId)
 		throw new Meteor.Error('invalid-user', '[methods] setEmail -> Invalid user');
```
Comment (rodrigok) [inline packages/rocketchat-lib/server/functions/setEmail.js]: Codacy Issue found: [Expected '{' and instead saw 'throw'.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2317368006/issues/source?bid=3199634&fileBranchId=3203539#l4)

File: packages/rocketchat-lib/server/functions/setEmail.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 RocketChat._setEmail = function(userId, email) {
 	email = s.trim(email);
-	if (!userId) {
+	if (!userId)
```
Comment (rodrigok) [inline packages/rocketchat-lib/server/functions/setEmail.js]: Codacy Issue found: [Expected { after 'if' condition.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2317368006/issues/source?bid=3199634&fileBranchId=3203539#l3)

File: packages/rocketchat-lib/server/functions/setEmail.js [hunk 3]
```diff
@@ -13,7 +12,7 @@ RocketChat._setEmail = function(userId, email) {
 		throw new Meteor.Error('email-invalid', email + ' is not a valid e-mail');
 	}
 
-	const user = RocketChat.models.Users.findOneById(userId);
+	const user = RocketChat.models.Users.findOneById(userId)
```
Comment (rodrigok) [inline packages/rocketchat-lib/server/functions/setEmail.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2317368006/issues/source?bid=3199634&fileBranchId=3203539#l15)

**Comment by engelgabriel** [comment]
this is cool!

**Pull request closed without merging**